packages/shared/src/graphql/users.ts

```diff
@@ -423,3 +434,46 @@ export const UNSUBSCRIBE_PERSONALIZED_DIGEST_MUTATION = gql`
     }
   }
 `;
+
+interface ReadingDay {
+  date: string;
+  reads: number;
+}
+
+export const getReadingStreak30Days = async (
+  id: string,
+): Promise<ReadingDay[]> => {
+  const today = new Date();
+  const last30Days = subDays(today, 30);
```

Comment:
should we allow this to be passed as an argument rather than being hard-coded? I am thinking that we might want to just get the last 7 days here since we don't display more than that in the `ReadingStreakPopup` at the moment

Reply:
Plus one on the above.

Reply:
I was thinking of even just the last 14 days. But as per RFC we will pull 30 days.

Reply:
Introduced a second parameter to decide when should the query start.